tests/angular_component_examples/jsfmt.spec.js

```diff
@@ -1,2 +1,2 @@
+run_spec(__dirname, ["typescript"], { trailingComma: "none" });
 run_spec(__dirname, ["typescript"]);
```

Comment:
I change the test orders to help review, snap chang should only be `options` part